must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -191,9 +191,13 @@ for ns in $namespaces; do
         done
     done
 
+    # Add Ready nodes to the list
+    nodes=()
+
     # Collecting ceph prepare volume logs
-    for node in $(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | grep -w 'Ready' | awk '{print $1}'); do
+    for node in $(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}'); do
         printf "collecting prepare volume logs from node %s \n"  "${node}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+        nodes+=("$(node)")
```

Comment:
You shouldn't need to do this. Can't you just save the output of the command to `nodes` directly and just loop through it?

Reply:
This is the last remaining request. It should work roughly like this:
```
nodes=$(oc get nodes --no-headers | awk '/\yReady\y/{print $1}')
for node in $nodes ; do
    ...
done
```